Dockerfile

```diff
@@ -47,6 +47,11 @@ RUN curl https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
         llvm-${LLVM_VERSION} && \
     chmod -f +x /usr/lib/llvm-${LLVM_VERSION}/bin/*
 
+# Check and see Clang has not been rebuilt in more than five days if we are on the master branch and fail the build if so
+COPY clang-check.sh /
```

Comment:
was the forward slash intentional? I didn't see `$1` anywhere in the shell script.

Reply:
oh, it's copied, then removed, ok. sorry, I misunderstood that.

Why copy the shell script, run the copy, then remove it?

Reply:
Whoops, no it was not...

Reply:
> Why copy the shell script, run the copy, then remove it?

There is no point in having this in the image after the build (at least I don't think so).

Reply:
That's fine. Please add a comment to the dockerfile then (otherwise I'm going to forget that in the future, and the *why* is important).

Reply:
Done.